### PR TITLE
Bugfix/tesb/camel 3.11.1.x/appint 35136

### PR DESCRIPTION
**Pull request by xldai**

### CODE REVIEW

File: core/camel-componentdsl/src/generated/java/org/apache/camel/builder/component/dsl/SalesforceComponentBuilderFactory.java [hunk 1]
```diff
@@ -1267,6 +1267,23 @@ default SalesforceComponentBuilder instanceUrl(
...
+         * Group: security
+         * 
+         * @param jwtAudience the value to set
+         * @return the dsl builder
+         */
+        default SalesforceComponentBuilder jwtAudience(
+                java.lang.String jwtAudience) {
+            doSetProperty("jwtAudience", jwtAudience);
+            return this;
+        }
```
Comment (essobedo) [inline core/camel-componentdsl/src/generated/java/org/apache/camel/builder/component/dsl/SalesforceComponentBuilderFactory.java:1286]: It looks like some code regarding the worker pool is missing, get it from https://github.com/apache/camel/commit/9ef5654a1ed6a5ef72cb67c19fdb0b468b21e1f2#diff-dd5f4218fc0e6e69abc63722905c8cceaafd31a80a6bca9b47efde0b7b3fac2f

**Review by essobedo — CHANGES_REQUESTED** [review]
Sounds good 👍 , only one small remark to consider

File: core/camel-componentdsl/src/generated/java/org/apache/camel/builder/component/dsl/SalesforceComponentBuilderFactory.java (hunk 1 above)
Comment (xldai) [reply core/camel-componentdsl/src/generated/java/org/apache/camel/builder/component/dsl/SalesforceComponentBuilderFactory.java:1317]: Okay, merged the commit above into this PR.

**Review by essobedo — APPROVED** [review]
LGTM 👍

**Pull request merged**